src/Rector/MethodCall/UnaliasCollectionMethodsRector.php

```diff
@@ -70,7 +70,12 @@ private function updateMethodCall(MethodCall $methodCall): ?MethodCall
             $replacement = 'contains';
         } elseif ($this->isName($name, 'average')) {
             $replacement = 'avg';
-        } else {
+        } elseif ($this->isName($name, 'unlessEmpty')) {
+            $replacement = 'whenNotEmpty';
+        } elseif ($this->isName($name, 'unlessNotEmpty')) {
+            $replacement = 'whenEmpty';
+        }
+         else {
```

Comment:
might want to correct this

```suggestion
        } else {
```